django/core/management/commands/makemigrations.py

```diff
@@ -234,7 +234,14 @@ def handle_merge(self, loader, conflicts):
                 })
                 new_migration = subclass("%04i_merge" % (biggest_number + 1), app_label)
                 writer = MigrationWriter(new_migration)
-                with open(writer.path, "wb") as fh:
-                    fh.write(writer.as_string())
-                if self.verbosity > 0:
-                    self.stdout.write("\nCreated new merge migration %s" % writer.path)
+
+                if not self.dry_run:
+                    with open(writer.path, "wb") as fh:
```

Comment:
`# Write the migrations file to the disk.`
and something like
`# Alternatively, makemigrations --dry-run --verbosity 3 will output the merge migrations file to stdout rather than saving the file to the disk.`